.github/workflows/ci.yml

```diff
@@ -97,7 +97,9 @@ jobs:
     - name: Setup dotnet # Use setup dotnet action as Windows-2019 image no longer contains .net5.0
       uses: actions/setup-dotnet@v3
       with:
-        dotnet-version: '5.0.x'
+        dotnet-version: |
+          3.1.x
+          5.0.x
```

Comment:
pr description says they removed 3.1.x, but you are adding it?

Reply:
yep, they removed it from the image, so, we need to add it through the action.